src/widgets/Grabber.h

```diff
@@ -46,15 +46,13 @@ class GrabberEvent:public wxCommandEvent
    GrabberEvent(wxEventType type = wxEVT_NULL,
                 wxWindowID winid = 0,
                 const wxPoint& pt = wxDefaultPosition)
-   : wxCommandEvent(type, winid)
+   : wxCommandEvent(type, winid), mPos(pt)
```

Comment:
It's an optimization. Initialization lists are a little bit faster than assigning the value in the constructor body. And as one value already uses the initialization list, I simply moved that one over there too. ;)